app/renderer/js/components/handle-external-link.ts

```diff
@@ -1,15 +1,16 @@
-import { ipcRenderer, remote } from 'electron';
+import { ipcRenderer, remote, NewWindowEvent } from 'electron';
```

Comment:
Use Electron.NewWindowEvent instead of importing.

Reply:
Done